week2/stronghamsik/main.js

```diff
@@ -0,0 +1,146 @@
+const $ = (selector) => document.querySelector(selector);
+
+let burgerList = [];
+let accumulatedAmount = 0;
+
+function clickBurgerCard(){
+  const cardList = $('.mac__burger');
+  const shoppingList = $('.shopping__list');
+  // const cart = $('shopping__list')
+  cardList.addEventListener('click',(e)=>{  
+    const burger = e.target.closest('article');
```

Comment:
`closest` 사용 정말 좋은 방법인 것 같습니다! 배워가요! 그런데 카드 컨테이너 자체가 클릭했을 때는 실행이 안되도록 막아야 할 것 같아요! 그리고 이 html에선 사실 그럴 일 없겠지만 더 복잡한 뷰에선 상위에 또 다른 article 태그가 있을지 모를 일이어서 이럴 때 웬만하면 태그 이름보다 클래스를 선택자로 써주는 것이 좋아요!
```suggestion
    const burger = e.target.closest('burger__card');
    if(!burger) return;
```